pkg/tid/cli/display/table.go

```diff
@@ -76,6 +77,15 @@ func WriteTimesheetsTable(sheets []types.Timesheet, writer io.Writer) {
 	table.Render()
 }
 
+// getTimeInRightFormat prints the time using the format specified in config file.
+func getTimeInRightFormat(duration time.Duration, timeFormat string) string {
+	if (timeFormat == "decimal") {
```

Comment:
in go, parentheses on `if`s are not necessary, and often lead to a cleaner syntax. I have a feeling [go's formatting tool](https://golang.org/cmd/gofmt/) may deal with these? if not, I recommend looking at it anyway! 😃